accounts/views.py

```diff
@@ -14,10 +27,10 @@ def sign_up_view(request):
             return redirect('accounts:account')
     else:
         registration_form = SignUpForm()
-
     return render(request, 'signup.html', {'registration_form': registration_form})
 
 
 @login_required
 def account_view(request):
+    """ Account UI - work in progress """
```

Comment:
if it's in progress why the PR isn't [WIP]?

Reply:
I added the page so that the UI can work, but it doesn't have content yet(that will be added later). Should I remove the work in progress line or mark this as work in progress until all the rest of the features are completed?

Reply:
You should mention the specific issue for that...

Reply:
Should I update the comment in the file with the issue? Or in the PR/commit description? I only added comments here, but haven't done anything with this view in any of these commits

Reply:
In the file

Reply:
Done